xml/remote_administration.xml

```diff
@@ -258,16 +258,15 @@ nodenames[A-B]          attr[=value],attr[=value],...
   <para>
    <emphasis>MUNGE</emphasis> allows for secure communications between
    different machines which share the same secret key. The most common
-   usecase is the <emphasis>Slurm</emphasis> workload manager, which
+   use case is the <emphasis>Slurm</emphasis> workload manager, which
    uses MUNGE for the encryption of its messages. Another use case is
-   authentication for the parallel shell
-   <emphasis>mrsh</emphasis>.
+   authentication for the parallel shell <emphasis>mrsh</emphasis>.
   </para>
   <sect2>
    <title>Setting up MUNGE authentication</title>
    <para>
     MUNGE uses UID/GID values to uniquely identify and authenticate users. Thus,
-    you must take care that users who will authenicate across a network have
+    you must take care that users who will authenticate across a network have
     unified UIDs and GIDs.
```

Comment:
I had already fixed "authenicate". 

I disagree re the replacement of unified with unique -- the point is that they must **not** be unique, they must be unified -- the same -- across all nodes. Unique GIDs are what this is instructing us to avoid.

Reply:
Ah--right. But still "unified" is kinda bad at communicating that. A bit longer but imo clearer:

```suggestion
    the matching UIDs and GIDs across all nodes.
```